data/tlg0540/tlg008/__cts__.xml

```diff
@@ -2,14 +2,13 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0540" projid="greekLit:tlg008" urn="urn:cts:greekLit:tlg0540.tlg008" xml:lang="grc">
             
    <ti:title xml:lang="eng">Accusation of Calumny</ti:title>
-   <ti:title xml:lang="eng">Accusation of Calumny Against Fellow-Members of a Society</ti:title>
             
    <ti:edition projid="greekLit:perseus-grc2" urn="urn:cts:greekLit:tlg0540.tlg008.perseus-grc2" workUrn="urn:cts:greekLit:tlg0540.tlg008" xml:lang="grc">
                 
       <ti:label xml:lang="eng">Accusation of Calumny</ti:label>
                 
-      <ti:description xml:lang="eng">Lysias with an English translation by W.R.M. Lamb, M.A.; Cambridge, MA, Harvard University Press; London, William Heinemann Ltd. 1930.</ti:description>
-                
+      <ti:description xml:lang="eng">Lysias. Lamb, W.R.M., editor. Cambridge, MA, Harvard University Press; London, William Heinemann Ltd. 1930.</ti:description>
```

Comment:
colon after cities, comma before pub year

Reply:
hey @lcerrato this is the same punctuation format from the last batch of committed files from Lysias, but I can go back and change those at the end.

Reply:
I think I changed those manually already.

Reply:
@lcerrato  of course you did. :)

Reply:
@AlisonBabeu I have to do something around here!!

Reply:
then @lcerrato onwards!